generator/committee_readme.tmpl

```diff
@@ -24,7 +24,7 @@ The [charter]({{.CharterLink}}) defines the scope and governance of the {{.Name}
 
 ## Members
 {{ range .Leadership.Chairs }}
-* {{.Name}} (**[@{{.GitHub}}](https://github.com/{{.GitHub}})**){{if .Company}}, {{.Company}}{{end}}
+* {{.Name}} (**{{if .Slack}}GitHub: {{end}}[@{{.GitHub}}](https://github.com/{{.GitHub}}){{if .Slack}}, Slack: [@{{.Slack}}](https://kubernetes.slack.com/team/{{.Slack}}){{end}}**){{if .Company}}, {{.Company}}{{end}}
```

Comment:
Do those links work? With the face that "names" are a non-unique space, would we need tie ins to https://github.com/kubernetes/community/blob/master/communication/slack-config/users.yaml to prevent folks from DMing the wrong person?

Reply:
They are not unique =/  the `users.yaml` file is our workaround.

Reply:
Ugh.  Is the "member ID" thing (eg: for me "U6UB5V4TX" instead of "tpepper") unique?  Both https://kubernetes.slack.com/team/U6UB5V4TX and https://kubernetes.slack.com/team/tpepper seem to drop to the same user profile page where there's an option to message.

If so, it seems then we'd need sigs.yaml to use that UID and have the parser populate the Person struct with both the UID and the human readable name, with the generator using the human readable name as the anchor which hyperlinks to an url using the UID.

Reply:
Yup, thats the unique bit. We keep track of that in [users.yaml](https://github.com/kubernetes/community/blob/master/communication/slack-config/users.yaml) for use with usergroups

Reply:
I'll refactor to leverage the users.yaml content.